### ISSUE TITLE
'Capture Page Screenshot' does not fail if it cannot save the screenshot

### ISSUE THREAD

**Issue by mulmschneider** [robotframework/SeleniumLibrary#322]
'Capture Page Screenshot' does not tell you when it failed to create/save a screenshot - if you for example pass it a path with a nonexistant directory, the keyword will pass without any indication that something is wrong. 

Example code:

```
*** Settings ***
Library           Selenium2Library
*** Test Cases ***
Create Screenshot
   Open Browser  https://google.com
   Capture Page Screenshot  doesnotexist/selenium.png
   Close Browser
```

(I ran into this because I missed the fact that the screenshots are saved relative to the log file, and spent quite some time before I could track my issue down to the '-l' parameter of pybot)

**Comment by ombre42** [comment]
Thanks for reporting this. The method we call returns a bool to indicate success/failure, but we are not checking the return value.

```
# Selenium's relavent code
png = self.get_screenshot_as_png()
        try:
            with open(filename, 'wb') as f:
                f.write(png)
        except IOError:
            return False
        finally:
            del png
        return True
```

**Comment by mgrijalva** [comment]
In this case the return value should be checked, and raise an exception if it is False?

**Comment by just-be-dev** [comment]
I've wrapped everything in if statements, but there isn't a whole lot of information I can give. All I know is that if failed, not why it failed. 

I've also addressed #324 so hopefully that should alleviate most of the failures. I just need to add some tests for this and hopefully I can close out this issue.

**Comment by just-be-dev** [comment]
This issue is being addressed in #381 and is slated for release in 1.7.0.
